bleachbit/CLI.py

```diff
@@ -233,7 +233,7 @@ def process_cmd_line():
         for wipe_path in args:
             logger.info('Wiping free space in path: %s', wipe_path)
             import bleachbit.FileUtilities
-            for ret in bleachbit.FileUtilities.wipe_path(wipe_path):
+            for _ in bleachbit.FileUtilities.wipe_path(wipe_path):
```

Comment:
@monosans 

This happened to me before. On line 26, we import `_()` function, which is used for localization, but here on line 236 in the local scope of this function, it's redefined as a dummy variable, so you get the `UnboundLocalError: local variable '_' referenced before assignment` seen in the AppVeyor log. We need to be careful to distinguish the dummy variable and the gettext localization function.